### DISCUSSION TITLE
目次の表示位置を高くしたら見やすくなるのかもしれません

### DISCUSSION THREAD

**Discussion by yamatai12** [increments/qiita-discussions#805] [要望・アイデア・不具合]
<img width="384" alt="image" src="https://github.com/increments/qiita-discussions/assets/123088250/7b948921-1d53-46e6-97ef-6615df4aab61">

いつもお世話になっています！

目次の表示位置の高さがもう少し高くなると目次が見やすくなるかなと思うのですがいかがでしょうか？

Zennは目次の表示位置がQiitaさんより高い位置にあります。

以上よろしくお願いします🙏

**Comment by tanaka-asami415** [comment]
@yamatai12 
リクエストありがとうございます！
直近の開発優先度から、目次位置については直近でアップデートする予定はないのですが
今後の改善の参考にさせていただきます🙇
